src/main/kotlin/racingcar/MovingJudgeRule.kt

```diff
@@ -0,0 +1,34 @@
+package racingcar
+
+enum class MovingJudgeRule(
+    private val expression: (Int) -> (Boolean),
+    private val strategy: MovingStrategy
+) {
+
+    ADVANCE_RULE(
+        expression = { number -> number >= FORWARD_STRATEGY_BOUND },
+        strategy = MovingStrategy.ADVANCE,
+    ),
+    STOP_RULE(
+        expression = { false },
```

Comment:
아래의 표현이 더 올바른 조건식이지 않을까요? 🤔
```suggestion
        expression = { it < FORWARD_STRATEGY_BOUND },
```

Reply:
해당 부분은 제가 실수했네요 😢 리뷰어님의 코멘트 남겨주신 코드가 맞아요~!